# circos_plots

#cc16_vs_A22

```bash
OutDir=analysis/genome_alignment/circos/vs_AS22/cc16
mkdir -p $OutDir
```
#Transform our .fa files with the genomes in .txt files (both the sample and the reference genomes.)

```bash
QueryGenome=$(ls repeat_masked/*/*/filtered_contigs/*_contigs_hardmasked.fa| grep 'cc16')
ProgDir=~/git_repos/tools/seq_tools/circos
python $ProgDir/fasta2circos.py --genome $QueryGenome --contig_prefix "cc16_" > $OutDir/query_genome.txt

Ref_genome=$(ls assembly/misc_publications/c.albicans/A22_Chromosomes_29.fasta | grep 'A22')
ProgDir=~/git_repos/tools/seq_tools/circos
python $ProgDir/fasta2circos.py --genome $Ref_genome --contig_prefix "A22_" > $OutDir/ref_genome.txt

cat $OutDir/query_genome.txt > $OutDir/query_ref_genome.txt
tac $OutDir/ref_genome.txt >> $OutDir/query_ref_genome.txt
```

```bash
OutDir=analysis/genome_alignment/circos/vs_AS22/cc16
Coords=$(ls analysis/genome_alignment/mummer/c.albicans/cc16/cc16_vs_A22/cc16_vs_A22_coords.tsv)
ProgDir=~/git_repos/scripts/alternaria/pathogen/genome_alignment
$ProgDir/nucmer_coords2circos.py --inp_coords $Coords --queery_id cc16 --ref_id A22 > $OutDir/query_vs_ref_links.txt
cat $OutDir/query_vs_ref_links.txt > $OutDir/query_vs_ref_links_edited.txt
```

A file showing contig orientations was made:

```bash
cat $OutDir/query_ref_genome.txt | cut -f3 -d ' ' | sed "s/$/; /g" | tr -d '\n' > $OutDir/query_contig_order.txt
  ProgDir=~/git_repos/tools/seq_tools/circos
  $ProgDir/find_contig_orientation.py --links_file $OutDir/query_vs_ref_links_edited.txt > $OutDir/query_vs_ref_contig_orientation.txt
```

Contig order was selected by taking the first line of that file and then also taking the reversed order of contigs using the command:

```bash
cat $OutDir/query_vs_ref_contig_orientation.txt | grep -A1 'Order of all seen contigs' | tail -n1 | sed "s/, /\n/g" > tmp.txt
cat $OutDir/query_ref_genome.txt | grep 'A22' | grep -w -v -f tmp.txt | cut -f3 -d ' '| tr -d '\n' | sed 's/A22/, A22/g'
# cat $OutDir/Ag_As_genome.txt | grep 'Ag' | cut -f3 -d ' ' | tr -d '\n' | sed 's/Ag/, Ag/g' >> tmp.txt

echo "Order of unseen cc16 contigs and remaining cc16 contigs"
cat $OutDir/query_ref_genome.txt | grep -w -v -f tmp.txt | cut -f3 -d ' '| tr -d '\n' | sed 's/cc16/, cc16/g' | sed 's/A22/, A22/g'
```



